com.unity.netcode.gameobjects/Runtime/Spawning/NetworkPrefabHandler.cs

```diff
@@ -252,23 +342,38 @@ internal uint GetSourceGlobalObjectIdHash(uint networkPrefabHash)
         /// <param name="position"></param>
         /// <param name="rotation"></param>
         /// <returns></returns>
-        internal NetworkObject HandleNetworkPrefabSpawn(uint networkPrefabAssetHash, ulong ownerClientId, Vector3 position, Quaternion rotation)
+        internal NetworkObject HandleNetworkPrefabSpawn(uint networkPrefabAssetHash, ulong ownerClientId, Vector3 position, Quaternion rotation, FastBufferReader instantiationDataReader = default)
+        {
+            NetworkObject networkObjectInstance = instantiationDataReader.IsInitialized
```

Comment:
nit: We prefer not to call functions from within a ternary. It's a style guide thing to make the code a bit easier to read.